content/en/docs/tasks/observability/metrics/lightstep/index.md

```diff
@@ -0,0 +1,230 @@
+* A good understanding of Kubernetes
+
+## Configure the Collector
+
+You use a Kubernetes ConfigMap file to configure the Collector to scrape Prometheus metrics and a deployment file to deploy it to Kubernetes.
+
+1. Create an `otel-collector-configmap.yaml` file by copying the following code.
+
+    {{< text yaml >}}
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: otel-collector-conf
+    data:
+      otel-collector-config.yaml: |
+        receivers:
+          prometheus:
+            config:
+              scrape_configs:
+                - job_name: 'otel-collector'
+                  scrape_interval: 5s
+                  static_configs:
+                    - targets: ['0.0.0.0:8888']
+                - job_name: "istio"
+                  scrape_interval: 5s
+                  metrics_path: "/stats/prometheus"
+                  kubernetes_sd_configs:
+                    - role: "pod"
+                  relabel_configs:
+                  // add labels
+
+       processors:
+          batch:
+
+        exporters:
+            logging:
+            loglevel: debug
+          otlp:
+            endpoint: ingest.lightstep.com:443
+            headers:
+                    "lightstep-access-token": "{LS_ACCESS_TOKEN}"
+
+        service:
+          telemetry:
+            logs:
+              level: "debug"
+          pipelines:
+            metrics:
+              receivers: [prometheus]
+              processors: [batch]
+              exporters: [logging,otlp]
+    {{< /text >}}
+
+1. Create an `otel-collector-deployment.yaml` file by copying the following code.
+
+    {{< text yaml >}}
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: otel-collector
+      labels:
+        app: otel-collector
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: otel-collector
+      template:
+        metadata:
+          labels:
+            app: otel-collector
+        spec:
+          serviceAccountName: otel-collector
+          containers:
+          - name: otel-collector
+            image: otel/opentelemetry-collector-contrib:latest
+            args:
+            - "--config=/conf/otel-collector-config.yaml"
+            ports:
+            - containerPort: 55681
+            env:
+            - name: LS_ACCESS_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: lightstep-access-token
+                  key: {LS_ACCESS_TOKEN}
+
+            volumeMounts:
+            - name: otel-collector-config-vol
+              mountPath: /conf
+          volumes:
+          - configMap:
+              name: otel-collector-conf
+            name: otel-collector-config-vol
+    {{< /text >}}
+
+## Configure Istio Operator
+
+Create an `istio-operator.yaml` file by copying the following code.
```

Comment:
Is 'istio-operator.yaml' here used in the guide? A quick search did not return any results.

Reply:
I'm not sure what you mean by "used in the guide?" Do you mean elsewhere in Istio docs? It's used in this topic to configure the k8s operator for Istio.

Reply:
> "It's used in this topic to configure the k8s operator for Istio."
Which command lines in this guide use 'istio-operator.yaml'?

Reply:
Thanks - I realized what he was asking and agreed, so I sent it back to the engineering team. They are working on updates.Thanks for catching that - I definitely should have, but with this rebrand, I've been 🤪!